zappa/cli.py

```diff
@@ -437,6 +438,22 @@ def positive_int(s):
             'shell', parents=[env_parser], help='A debug shell with a loaded Zappa object.'
         )
 
+        ##
+        # Python Settings File
+        ##
+        settings_parser = subparsers.add_parser(
+            'save-python-settings-file', parents=[env_parser],
+            help='Generate & save the Zappa settings Python file for docker deployments'
+        )
```

Comment:
I opted to make this new CLI method very explicit by calling it `save-python-settings-file` instead of just `settings`. I did this because there is already the concept of the `zappa_settings.json` / `zappa_settings.yml` file, so wanted to make it clear that we are talking about hte `zappa_settings.py` file that gets used by the `handler.py`

Reply:
Would `--handler-settings-file` be a better name? (you can actually have zappa_settings.py as the configuration file too)

Reply:
Interesting, @jneves reviewing my notes from before: `handler.py` looks for a `zappa_settings.py` file as per https://github.com/Miserlou/Zappa/blob/ba20c850eeca00edd6ea39fda1ab976cfee193ea/zappa/handler.py#L59-L65

If we were to name the file something other than `zappa_settings.py`, i think that wouldn't work? Or are you just suggesting we name the CLI command `save-handler-settings-file` instead of `save-python-settings-file`?